docs/getting-started/getting-started-with-notebooks.rst

```diff
@@ -0,0 +1,164 @@
+Notebooks
+=========
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note:: Dockstore Notebooks is currently in preview mode. Also note that the screenshots below were taken on our staging site.
+
+Tutorial Goals
+--------------
+
+- Get familiarized with the concept of notebooks on Dockstore
+- Learn the basics of creating a notebook using a .dockstore.yml file
+- Learn how to register a notebook
+- Publish your notebook to Dockstore
+
+Notebooks are documents that can include code, text, images, and other media resources. They can be executed in notebook environments like Google Colab and JupyterLab.
+This document will outline what is needed to register, update, and publish a notebook onto Dockstore. Please keep in mind that notebook functionality is currently in preview mode.
+
+Overview
+--------
+
+In this tutorial, you will register a notebook using the Dockstore GitHub App, which automatically syncs your notebook from GitHub to Dockstore. More information about the Dockstore GitHub App can be found :doc:`here </getting-started/github-apps/github-apps-landing-page>`.
+
+Dockstore Notebooks takes advantage of Google Colab's integration with GitHub. The integration allows you to develop your notebook in Google Colab and push your changes to GitHub from Google Colab. With the Dockstore GitHub App installed on your repository, any edits made to your notebook from Google Colab will automatically be synced to Dockstore.
```

Comment:
Suggest that we changed "made" to "saved" in the last sentence, to reinforce that the changes won't be synced until they're actually saved to GitHub.